scripts/build

```diff
@@ -148,6 +155,8 @@ if [[ ! -d $STEP_DIR ]]; then
     $MAKE install-gcc install-target-libgcc || exit 34
   ) || err $?
 
+  export CXXFLAGS="$ORIGINAL_CXXFLAGS"
```

Comment:
musl uses GCC to compile and it doesn't recognize -fbracket-depth=512 and causes compiles to fail.